### DISCUSSION TITLE
Using fader to control Kontakt instruments

### DISCUSSION THREAD

**Discussion by onlinejoel** [navelpluisje/reasonus-install#55] [General]
I would like to control CC in Kontakt. When I enter "MIDI mode" I thought I'd be able to control parameters in Kontakt. What is the correct way to control CC and other Kontakt instrument parameters using FP16 faders

**Comment by navelpluisje** [comment]
Hi Joel, When using the FaderPort as a controller it is disabled as a midi device. But the FaderPort creates multiple MIDI devices. One of them is Port 1. That one should be set to disabled in the MIDI settings of REAPER. There should also be a MIDI Mode device in the list. Can you set that one to enabled (I assume it is set to disabled as well) for both the in and output and then give it a try?

BTW, how do you enter MIDI mode on the FaderPort? You can toggle it do it by pressing both Shift buttons at the same time, so you do not have to reboot it every time.

Please let me know if it works out tis way

**Comment by onlinejoel** [comment]
Hey quick question regarding Faderport 16/Reasonus. I've tried everything
and I can't seem to solve. I recently reinstalled and everything seems to
be working fine other than the default keys (non-shift key press). When I
touch the "Click" it used toggle the metronome. Now it does nothing. Action
list doesn't detect anything when I press the same key. Shift F1-F8
function as normal. What am I missing?

On Fri, Oct 20, 2023 at 1:21 AM Navelpluisje ***@***.***>
wrote:

> Hi Joel, When using the FaderPort as a controller it is disabled as a midi
> device. But the FaderPort creates multiple MIDI devices. One of them is
> Port 1. That one should be set to disabled in the MIDI settings of REAPER.
> There should also be a MIDI Mode device in the list. Can you set that one
> to enabled (I assume it is set to disabled as well) for both the in and
> output and then give it a try?
>
> BTW, how do you enter MIDI mode on the FaderPort? You can toggle it do it
> by pressing both Shift buttons at the same time, so you do not have to
> reboot it every time.
>
> Please let me know if it works out tis way
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/navelpluisje/reasonus-install/discussions/55#discussioncomment-7334948>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASX2QT6W5DWYI3PZLJ2DSG3YAIC6VAVCNFSM6AAAAAA6H734FWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TGMZUHE2DQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> .com>
>


-- 

Joel Schwartz <https://joelschwartz.ca/>




Producer <http://joelschwartz.ca/>

Composer <https://filmguitars.com/>

Guitarist

[image: link]

https://joelschwartz.ca
[image: link]

https://filmguitars.com
[image: envelope]

***@***.***
[image: map-marker]

2775 Dundas St. West, Toronto, Ontario M6P 1Y4
***@***.***,-79.4800088,15.43z/data=!4m6!3m5!1s0x863d4e2b9ec533cb:0x9228abf2f581513b!8m2!3d43.7182412!4d-79.3780581!16s%2Fg%2F11p_1ywf86?entry=ttu&g_ep=EgoyMDI1MDcwOS4wIKXMDSoASAFQAw%3D%3D>



[image: facebook] <https://www.facebook.com/itsjoel/> [image: instagram]
<https://www.instagram.com/joel.schwartz>

**Comment by onlinejoel** [comment]
To add a bit more detail - the firmware is 3.74 and Reasonus is current
version

On Wed, Aug 27, 2025 at 11:35 AM Joel Schwartz ***@***.***> wrote:

> Hey quick question regarding Faderport 16/Reasonus. I've tried everything
> and I can't seem to solve. I recently reinstalled and everything seems to
> be working fine other than the default keys (non-shift key press). When I
> touch the "Click" it used toggle the metronome. Now it does nothing. Action
> list doesn't detect anything when I press the same key. Shift F1-F8
> function as normal. What am I missing?
>
> On Fri, Oct 20, 2023 at 1:21 AM Navelpluisje ***@***.***>
> wrote:
>
>> Hi Joel, When using the FaderPort as a controller it is disabled as a
>> midi device. But the FaderPort creates multiple MIDI devices. One of them
>> is Port 1. That one should be set to disabled in the MIDI settings of
>> REAPER. There should also be a MIDI Mode device in the list. Can you set
>> that one to enabled (I assume it is set to disabled as well) for both the
>> in and output and then give it a try?
>>
>> BTW, how do you enter MIDI mode on the FaderPort? You can toggle it do it
>> by pressing both Shift buttons at the same time, so you do not have to
>> reboot it every time.
>>
>> Please let me know if it works out tis way
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/navelpluisje/reasonus-install/discussions/55#discussioncomment-7334948>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/ASX2QT6W5DWYI3PZLJ2DSG3YAIC6VAVCNFSM6AAAAAA6H734FWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TGMZUHE2DQ>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> <navelpluisje/reasonus-install/repo-discussions/55/comments/7334948@
>> github.com>
>>
>
>
> --
>
> Joel Schwartz <https://joelschwartz.ca/>
>
>
>
>
> Producer <http://joelschwartz.ca/>
>
> Composer <https://filmguitars.com/>
>
> Guitarist
>
> [image: link]
>
> https://joelschwartz.ca
> [image: link]
>
> https://filmguitars.com
> [image: envelope]
>
> ***@***.***
> [image: map-marker]
>
> 2775 Dundas St. West, Toronto, Ontario M6P 1Y4
> ***@***.***,-79.4800088,15.43z/data=!4m6!3m5!1s0x863d4e2b9ec533cb:0x9228abf2f581513b!8m2!3d43.7182412!4d-79.3780581!16s%2Fg%2F11p_1ywf86?entry=ttu&g_ep=EgoyMDI1MDcwOS4wIKXMDSoASAFQAw%3D%3D>
>
>
>
> [image: facebook] <https://www.facebook.com/itsjoel/> [image: instagram]
> <https://www.instagram.com/joel.schwartz>
>
>
>

-- 

Joel Schwartz <https://joelschwartz.ca/>




Producer <http://joelschwartz.ca/>

Composer <https://filmguitars.com/>

Guitarist

[image: link]

https://joelschwartz.ca
[image: link]

https://filmguitars.com
[image: envelope]

***@***.***
[image: map-marker]

2775 Dundas St. West, Toronto, Ontario M6P 1Y4
***@***.***,-79.4800088,15.43z/data=!4m6!3m5!1s0x863d4e2b9ec533cb:0x9228abf2f581513b!8m2!3d43.7182412!4d-79.3780581!16s%2Fg%2F11p_1ywf86?entry=ttu&g_ep=EgoyMDI1MDcwOS4wIKXMDSoASAFQAw%3D%3D>



[image: facebook] <https://www.facebook.com/itsjoel/> [image: instagram]
<https://www.instagram.com/joel.schwartz>

**Reply by navelpluisje** [reply]
Hi Joel,
Did you install the version from here or ReaSonus Native?
This because I stopped updating and supporting this version. In the mean time i created ReaSonus Native. This is an extension written in C++ and not depending on CSI, with mostly the same features ad this version, but more stable and easier to maintain as it does not depend on other software (accept SWS and ImGui).

Also installing is now way easier as it is installable with ReaPack. If you do not use that, it's also fairly simple to install.

Documentation and info about ReaSonus Native can be found here: https://navelpluisje.github.io/reasonus/.

**Comment by onlinejoel** [comment]
I installed via ReaPack 0.2.0). I wonder if there is an issue with previous
install somehow?

On Wed, Aug 27, 2025 at 1:50 PM Navelpluisje ***@***.***>
wrote:

> Hi Joel,
> Did you install the version from here or ReaSonus Native?
> This because I stopped updating and supporting this version. In the mean
> time i created ReaSonus Native. This is an extension written in C++ and not
> depending on CSI, with mostly the same features ad this version, but more
> stable and easier to maintain as it does not depend on other software
> (accept SWS and ImGui).
>
> Also installing is now way easier as it is installable with ReaPack. If
> you do not use that, it's also fairly simple to install.
>
> Documentation and info about ReaSonus Native can be found here:
> https://navelpluisje.github.io/reasonus/.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/navelpluisje/reasonus-install/discussions/55#discussioncomment-14236060>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASX2QTYBYQBYZ36RONZKGO33PXVXPAVCNFSM6AAAAACE6TASXSVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTIMRTGYYDMMA>
> .
> You are receiving this because you authored the thread.Message ID:
> <navelpluisje/reasonus-install/repo-discussions/55/comments/14236060@
> github.com>
>


-- 

Joel Schwartz <https://joelschwartz.ca/>




Producer <http://joelschwartz.ca/>

Composer <https://filmguitars.com/>

Guitarist

[image: link]

https://joelschwartz.ca
[image: link]

https://filmguitars.com
[image: envelope]

***@***.***
[image: map-marker]

2775 Dundas St. West, Toronto, Ontario M6P 1Y4
***@***.***,-79.4800088,15.43z/data=!4m6!3m5!1s0x863d4e2b9ec533cb:0x9228abf2f581513b!8m2!3d43.7182412!4d-79.3780581!16s%2Fg%2F11p_1ywf86?entry=ttu&g_ep=EgoyMDI1MDcwOS4wIKXMDSoASAFQAw%3D%3D>



[image: facebook] <https://www.facebook.com/itsjoel/> [image: instagram]
<https://www.instagram.com/joel.schwartz>

**Comment by onlinejoel** [comment]
Hey there, thank you so much for your help. Getting most of my
functionality back now. There are some differences I'm noticing and I'm
wondering if you can help with another thing that's popped up.

Previously, I was able to trigger metronome on/off with the "Click" button
and I was able to program the functionality of the green forward/back
buttons. Is there a way to do this?

On Wed, Aug 27, 2025 at 1:54 PM Joel Schwartz ***@***.***> wrote:

> I installed via ReaPack 0.2.0). I wonder if there is an issue with
> previous install somehow?
>
> On Wed, Aug 27, 2025 at 1:50 PM Navelpluisje ***@***.***>
> wrote:
>
>> Hi Joel,
>> Did you install the version from here or ReaSonus Native?
>> This because I stopped updating and supporting this version. In the mean
>> time i created ReaSonus Native. This is an extension written in C++ and not
>> depending on CSI, with mostly the same features ad this version, but more
>> stable and easier to maintain as it does not depend on other software
>> (accept SWS and ImGui).
>>
>> Also installing is now way easier as it is installable with ReaPack. If
>> you do not use that, it's also fairly simple to install.
>>
>> Documentation and info about ReaSonus Native can be found here:
>> https://navelpluisje.github.io/reasonus/.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/navelpluisje/reasonus-install/discussions/55#discussioncomment-14236060>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/ASX2QTYBYQBYZ36RONZKGO33PXVXPAVCNFSM6AAAAACE6TASXSVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTIMRTGYYDMMA>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> <navelpluisje/reasonus-install/repo-discussions/55/comments/14236060@
>> github.com>
>>
>
>
> --
>
> Joel Schwartz <https://joelschwartz.ca/>
>
>
>
>
> Producer <http://joelschwartz.ca/>
>
> Composer <https://filmguitars.com/>
>
> Guitarist
>
> [image: link]
>
> https://joelschwartz.ca
> [image: link]
>
> https://filmguitars.com
> [image: envelope]
>
> ***@***.***
> [image: map-marker]
>
> 2775 Dundas St. West, Toronto, Ontario M6P 1Y4
> ***@***.***,-79.4800088,15.43z/data=!4m6!3m5!1s0x863d4e2b9ec533cb:0x9228abf2f581513b!8m2!3d43.7182412!4d-79.3780581!16s%2Fg%2F11p_1ywf86?entry=ttu&g_ep=EgoyMDI1MDcwOS4wIKXMDSoASAFQAw%3D%3D>
>
>
>
> [image: facebook] <https://www.facebook.com/itsjoel/> [image: instagram]
> <https://www.instagram.com/joel.schwartz>
>
>
>

-- 

Joel Schwartz <https://joelschwartz.ca/>




Producer <http://joelschwartz.ca/>

Composer <https://filmguitars.com/>

Guitarist

[image: link]

https://joelschwartz.ca
[image: link]

https://filmguitars.com
[image: envelope]

***@***.***
[image: map-marker]

2775 Dundas St. West, Toronto, Ontario M6P 1Y4
***@***.***,-79.4800088,15.43z/data=!4m6!3m5!1s0x863d4e2b9ec533cb:0x9228abf2f581513b!8m2!3d43.7182412!4d-79.3780581!16s%2Fg%2F11p_1ywf86?entry=ttu&g_ep=EgoyMDI1MDcwOS4wIKXMDSoASAFQAw%3D%3D>



[image: facebook] <https://www.facebook.com/itsjoel/> [image: instagram]
<https://www.instagram.com/joel.schwartz>

**Reply by navelpluisje** [reply]
Great to hear this. Both of the options are currently not implemented. 

How did you change the ffwd and rwd buttons? And what functionality did you assign to them?
I'm planning to add 8 extra function buttons (using right shift for the function buttons). Would that work for you instead of re-assiging the ffwd and rwd buttons? 
This because the idea for ReaSonus Native is to keep it as simple as possible and try to reduce the amount of settings and options

**Comment by onlinejoel** [comment]
Truth be told, I do not remember how I programmed them. I don't think
anything special - would have been native to Reaper. I assigned back and
forward to jump back/forward a bar (or similar) rather than scrub.

I hear you about keeping the interface clean, however, I for one would
really love to be able to customize and use the buttons that best suit my
workflow. Def missing click!

Let me know if I can help or provide more details.

On Thu, Sep 4, 2025 at 4:14 PM Navelpluisje ***@***.***>
wrote:

> Great to hear this. Both of the options are currently not implemented.
>
> How did you change the ffwd and rwd buttons? And what functionality did
> you assign to them?
> I'm planning to add 8 extra function buttons (using right shift for the
> function buttons). Would that work for you instead of re-assiging the ffwd
> and rwd buttons?
> This because the idea for ReaSonus Native is to keep it as simple as
> possible and try to reduce the amount of settings and options
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/navelpluisje/reasonus-install/discussions/55#discussioncomment-14311435>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASX2QTZC62XJD3IDTOPYA433RCMSPAVCNFSM6AAAAACE6TASXSVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTIMZRGE2DGNI>
> .
> You are receiving this because you authored the thread.Message ID:
> <navelpluisje/reasonus-install/repo-discussions/55/comments/14311435@
> github.com>
>


-- 

Joel Schwartz <https://joelschwartz.ca/>




Producer <http://joelschwartz.ca/>

Composer <https://filmguitars.com/>

Guitarist

[image: link]

https://joelschwartz.ca
[image: link]

https://filmguitars.com
[image: envelope]

***@***.***
[image: map-marker]

2775 Dundas St. West, Toronto, Ontario M6P 1Y4
***@***.***,-79.4800088,15.43z/data=!4m6!3m5!1s0x863d4e2b9ec533cb:0x9228abf2f581513b!8m2!3d43.7182412!4d-79.3780581!16s%2Fg%2F11p_1ywf86?entry=ttu&g_ep=EgoyMDI1MDcwOS4wIKXMDSoASAFQAw%3D%3D>



[image: facebook] <https://www.facebook.com/itsjoel/> [image: instagram]
<https://www.instagram.com/joel.schwartz>
